op-validator/pkg/validations/addresses.go

```diff
@@ -26,8 +26,8 @@ var addresses = map[uint64]map[string]common.Address{
 		VersionV180: common.HexToAddress("0x0a5bf8ebb4b177b2dcc6eba933db726a2e2e2b4d"),
 		// Bootstrapped on 03/02/2025 using OP Deployer.
 		VersionV200: common.HexToAddress("0x37739a6b0a3f1e7429499a4ec4a0685439daff5c"),
-		// Bootstrapped on 03/10/2025 using OP Deployer.
-		VersionV300: common.HexToAddress("0x5fd6dc82a0cbc0db40d18963d812487772113216"),
+		// Bootstrapped on 04/02/2025 using OP Deployer.
+		VersionV300: common.HexToAddress("0xd0d7f75561bb7105db2d34dc39b31f23578fb75b"),
```

Comment:
I think you might have the wrong impl addresses, for example the [`l1ERC721BridgeImpl` address](https://github.com/ethereum-optimism/superchain-registry/blob/934ea2dd22d160e076ebbecb310582049eec071d/validation/standard/standard-versions-sepolia.toml#L96) matches v2.0.0-rc.1
```
cast call 0xd0d7f75561bb7105db2d34dc39b31f23578fb75b 'l1ERC721BridgeImpl()(address)'
0x276d3730f219f7ec22274f7263180b8452B46d47
```

Reply:
Just updated, does that one have the correct rc addresses? Pulled from [https://github.com/ethereum-optimism/superchain-registry/blob/934ea2dd22d160e076eb[…]82049eec071d/validation/standard/standard-versions-sepolia.toml](https://github.com/ethereum-optimism/superchain-registry/blob/934ea2dd22d160e076ebbecb310582049eec071d/validation/standard/standard-versions-sepolia.toml#L8)